src/ResourceManager/EventHub/Commands.EventHub/Cmdlets/Namespace/SetAzureEventHubNamespace.cs

```diff
@@ -66,21 +67,39 @@ public class SetAzureEventHubNamespace : AzureEventHubsCmdletBase
         [Parameter(Mandatory = false,
             ValueFromPipelineByPropertyName = true,            
             HelpMessage = "Disable/Enable Namespace.")]
-        public Management.EventHub.Models.NamespaceState? State { get; set; }
+        public Models.NamespaceState? State { get; set; }
 
         [Parameter(Mandatory = false,
             ValueFromPipelineByPropertyName = true,
             HelpMessage = "Hashtables which represents resource Tag.")]
         public Hashtable Tag { get; set; }
 
+        /// <summary>
+        /// Indicates whether AutoInflate is enabled.
+        /// </summary>
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Indicates whether AutoInflate is enabled")]
+        public bool? IsAutoInflateEnabled { get; set; }
+
+        /// <summary>
+        /// Upper limit of throughput units when AutoInflate is enabled.
+        /// </summary>
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Upper limit of throughput units when AutoInflate is enabled, vaule should be within 0 to 20 throughput units.")]
+        public int? MaximumThroughputUnits { get; set; }
```

Comment:
@v-Ajnava same comments about `ValidateSet`, typo, and `AutoInflate` being disabled